packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -542,6 +545,25 @@ export default class New extends Component {
     })
   }
 
+  _probe = async (host, type) => {
+    const probeMethodFactories = {
+      hba: async hostId => ({
+        hbaDevices: await probeSrHba(hostId)::ignoreErrors(),
+      }),
+      zfs: async hostId => ({
+        zfsPools: await probeZfs(hostId)::ignoreErrors(),
+      }),
+    }
+    if (probeMethodFactories[type] !== undefined && host !== null) {
```

Comment:
Isn't `host` `undefined` when you haven't selected one yet?

Reply:
when `host` haven't ever been selected, its value is `undefined`
when `host` was selected and then cleared from the Select input, its value is `null`

Reply:
I see what you mean ... the test should be `host != null` instead of `host !== null`